### PR TITLE
Create Jenkinsfile for weekly mainnet tests - Closes #845

### LINKED ISSUE #845: Create Jenkinsfile for weekly mainnet tests
### Description

Create Jenkinsfile that runs Lisk Service against mainnet and triggers full indexing.

Create another Jenkinsfile that runs **regression** tests on the full blockchain index. 

Create another Jenkinsfile that runs **performance** tests on the full blockchain index. 

### Acceptance Criteria

- Every week a new deployment is done
- Regression tests are performed
- Development branch is used

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #845

### How was it solved?

- [x] Create Jenkinsfile that runs Lisk Service against Mainnet and triggers full indexing.
- [x] Create Jenkinsfile that runs regression tests on the full blockchain index.
- [x] Create Jenkinsfile that runs performance tests on the full blockchain index.

### How was it tested?
Jenkins

### CODE REVIEW

File: Jenkinsfile.deployment.mainnet [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+				ENABLE_JOB_MISSING_BLOCKS='true'
+				INDEX_MISSING_BLOCKS_RANGE='0'
+				ENABLE_TRANSACTION_STATS='true'
+				TRANSACTION_STATS_HISTORY_LENGTH_DAYS='366'
+			}
+			steps {
+				ansiColor('xterm') {
+					dir('./') {
+						sh 'make down'
+						sh 'make up-custom-core3-cd'
```
Comment (MichalTuleja) [inline Jenkinsfile.deployment.mainnet:37]: I think this script is already removed. `make up` with right params should do the job.

File: Jenkinsfile.performance.mainnet [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+def waitForHttp(url) {
+	waitUntil { script { return (checkHttp(url) == true) } }
+}
+
+pipeline {
+	agent none
+	triggers {
+		// Triggers every week at midnight on Sunday morning
+		cron('0 0 * * 0')
```
Comment (MichalTuleja) [inline Jenkinsfile.performance.mainnet:21]: Assuming the deployment is performed at 12:00pm, the test can be delayed one hour or so. To make sure the process works stable and there are no pending background tasks.

File: Jenkinsfile.regression.mainnet [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+def waitForHttp(url) {
+	waitUntil { script { return (checkHttp(url) == true) } }
+}
+
+pipeline {
+	agent none
+	triggers {
+		// Triggers every week at midnight on Sunday morning
+		cron('0 0 * * 0')
```
Comment (MichalTuleja) [inline Jenkinsfile.regression.mainnet:21]: Same as above.

**Review by MichalTuleja — CHANGES_REQUESTED** [review]
Few things need to be clarified. Are the tests already run on Jenkins?

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**